apps/console/src/features/server-configurations/pages/governance-connectors.tsx

```diff
@@ -81,9 +81,10 @@ export const GovernanceConnectorsPage: FunctionComponent<GovernanceConnectorsPag
         loadCategoryConnectors();
     }, []);
 
+    const path = history.location.pathname.split("/");
```

Comment:
Add missing types to `path` & `categoryId`.

```suggestion
    const path = history.location.pathname.split("/");
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/2987/commits/58a9084e0d55d1f34b7934f09b6fa06c328c22af